src/main/java/de/tum/in/www1/artemis/web/rest/plagiarism/PlagiarismCaseResource.java

```diff
@@ -66,6 +69,31 @@ public ResponseEntity<List<PlagiarismCase>> getPlagiarismCasesForCourseForInstru
         return getPlagiarismCasesResponseEntity(plagiarismCases);
     }
 
+    /**
+     * Retrieves all plagiarism cases related to an exam for the instructor view.
+     *
+     * @param courseId the course id to which the exam belongs
+     * @param examId the id of the exam
+     * @return all plagiarism cases of the course
+     */
+    @GetMapping("courses/{courseId}/exams/{examId}/plagiarism-cases/for-instructor")
+    @PreAuthorize("hasRole('INSTRUCTOR')")
+    public ResponseEntity<List<PlagiarismCase>> getPlagiarismCasesForExamForInstructor(@PathVariable long courseId, @PathVariable long examId) {
+        log.debug("REST request to get all plagiarism cases for instructor in exam with id: {}", examId);
+        Course course = courseRepository.findByIdElseThrow(courseId);
+        if (!authenticationCheckService.isAtLeastInstructorInCourse(course, userRepository.getUserWithGroupsAndAuthorities())) {
```

Comment:
```suggestion
        if (!authenticationCheckService.isAtLeastInstructorInCourse(course, null)) {
```
There is no benefit of fetching the user here if it is not used anywhere else afterwards.